code/modules/jobs/job_types/warden.dm

```diff
@@ -1,13 +1,13 @@
 /datum/job/warden
-	title = "Warden"
+	title = "Jailer"
```

Comment:
Suggest changing this to "Gaoler"
Which is the archaic way of saying Jailer.

Reply:
harder to understand what the role actually is from 'gaoler' vs 'jailer' to 'warden' so i'm not gonna do that, neat suggestion though